src/core/server/server.api.md

```diff
@@ -1,1764 +1,1787 @@
-## API Report File for "kibana"
```

Comment:
I don't know why it decided to re-generate the whole file. it gives~70% changes in the PR 😄

Reply:
Yeah I really don't understand what's going on here? Did the line endings change to windows format or something like that?